modules/msk/msisensor-ct/scan/main.nf

```diff
@@ -0,0 +1,33 @@
+process MSISENSOR_CT_SCAN {
+    tag "$meta.id"
+    label 'process_low'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        ghcr.io/msk-access/msisensor-ct:1.0 }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*.list"), emit: list
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    msisensor-ct \\
+        scan \\
+        -d $fasta \\
+        -o ${prefix}.msisensor_scan.list \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        msisensor-ct: \$(msisensor-ct 2>&1 | sed -nE 's/Version:\\sv([0-9]\\.[0-9])/\\1/ p')
+    END_VERSIONS
+    """
+}
```

Comment:
add stub test here. See here: https://github.com/mskcc-omics-workflows/modules/blob/73b78e1dfa2121a120b8c681162525d428ef8bb7/modules/msk/gbcms/main.nf#L40-L47. We're faking the output files for the empty run of the module that we do here: https://github.com/mskcc-omics-workflows/modules/blob/73b78e1dfa2121a120b8c681162525d428ef8bb7/tests/modules/msk/gbcms/main.nf#L7-L24